sdcard/FREQMAN/CB_UHF_AUSTRALIA.txt

```diff
@@ -0,0 +1,80 @@
+f=476425000,m=NFM,bw=8k5,d=CH 1 Repeater
```

Comment:
Something is odd with the bandwidth values...  Maybe they should all be 8k5?

Reply:
That there is an Excel autofill error! Thanks for this, fixed in the PR.